firedrake/utility_meshes.py

```diff
@@ -64,7 +64,7 @@
 ]
 
 
-distribution_parameters_noop = {"partition": False,
+distribution_parameters_noop = {"partition": True,
```

Comment:
please remove `_noop`

Reply:
Renamed it `distribution_parameters_no_overlap`.